pages/solidity-api/ArxProjectEnrollmentManager.md

```diff
@@ -107,7 +109,8 @@ unless we know the address ahead of time, hence we use CREATE2 which allows us t
 ```solidity
 function setTransferPolicy(contract ITransferPolicy _transferPolicy) external
 ```
-
+//TODO: also, in the architecture diagram, which one is ArxProjectEnrollmentManager?
+// TODO: i thought transfer policy was set by the current item owner?
```

Comment:
Was confused a little bit about which role is the ArxEnrollmentManager? Is this a TSM?

![Screenshot 2023-12-22 at 8 23 47 PM](https://github.com/arx-research/ers-docs/assets/25701682/9c0b93db-47bf-40c8-814d-4f717e3de88b)
Was unsure which in this diagram, this `ArxProjectEnrollmentManager` is.